test/integration/common.js

```diff
@@ -14,31 +14,9 @@ exports.Arrow = Arrow;
 exports.server = server;
 exports.connector = server.getConnector('appc.mongo');
 
-before(function (next) {
-	this.timeout(30000);
-	MongoClient.connect(mongoURL, function didConnect(err, db) {
-		if (err) {
-			return console.error(err);
-		}
-		db.collection('super_post').insert([
-			{Hello: 'world!', Foo: 2},
-			{Hello: 'sun!', Foo: 5},
-			{divergentDocument: true},
-			{Hello: 'sky!', Foo: 7},
-			{Hello: 'Earth!', Foo: 1},
-			{Hello: 'birds!', Foo: 3},
-			{How: 'are you today?!', Foo: 3}
-		], function (err) {
-			next(err);
-		});
-	});
-});
-
 before(function before(next) {
 	this.timeout(30000);
```

Comment:
I think we can go without it, because it's already set in the Gruntfile.js. I know that it was already here, but you can remove it along with the other changes that I requested.